extensions/wikia/ArticleVideoMVP/ArticleVideoMVP.hooks.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class ArticleVideoMVPHooks {
```

Comment:
I'm not sure if extension name should contain version name like `MVP`. If we continue development there we'd need to rename it. Which isn't for free.